### PR TITLE
[feature] added PPG quality assessment functionality

### PR DESCRIPTION
**Pull request by peterhcharlton**
# Description

This PR aims at adding a new feature: PPG quality assessment. PPG quality assessment functionality has been added, and two initial quality assessment techniques have been included.

# Proposed Changes

- Added ppg_quality.py
- Adjusted ppg_process.py to include quality assessment in the default processing pipeline
- Added details of the PPG quality assessment methods to ppg_methods.py

# Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#structure-and-code) file.
- [x] My PR is targeted at the **dev branch** (and not towards the master branch).
- [ ] I ran the [CODE CHECKS](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#run-code-checks) on the files I added or modified and fixed the errors.
- [ ] I have added the newly added features to **News.rst** (if applicable)

### CODE REVIEW

**Review by danibene — COMMENTED** [review]
Hi @peterhcharlton , this is a really nice addition, thanks for your contribution! I think the tests may be failing because of the deletion of "PPG_Peaks" from the signals dataframe.

**Review by danibene — APPROVED** [review]

File: neurokit2/ppg/ppg_process.py [hunk 1]
```diff
@@ -94,13 +98,22 @@ def ppg_process(
...
+        **methods["kwargs_quality"]
+    )
+
     # Prepare output
     signals = pd.DataFrame(
         {
             "PPG_Raw": ppg_signal,
             "PPG_Clean": ppg_cleaned,
             "PPG_Rate": rate,
-            "PPG_Peaks": peaks_signal["PPG_Peaks"].values,
```
[the remark this answers is not in the document]
Comment (DominiqueMakowski) [reply neurokit2/ppg/ppg_process.py:103]: This should probably not be deleted

[the remark this answers is not in the document]
Comment (danibene) [reply neurokit2/ppg/ppg_process.py:103]: I added it back I think, not sure why it’s not showing up here: https://github.com/peterhcharlton/NeuroKit/commit/1cd76e49d1dbb7fb278378351d767fb7eeb7602b

[the remark this answers is not in the document]
Comment (DominiqueMakowski) [reply neurokit2/ppg/ppg_process.py:103]: Thanks Dani! Let's merge then :)

**Pull request merged**